src/pages/AllTodos.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react' ;
```

Comment:
`AllTodos`, `CompletedTodos` and `InProgressTodos` are basically the same component with different selectors. Use 1 comonent with a prop instead